app/controllers/resources_controller.rb

```diff
@@ -4,7 +4,7 @@ class ResourcesController < ApplicationController
   def index
     category_id = params.require :category_id
     relation = resources.joins(:categories).joins(:address)
-                        .where('categories.id' => category_id)
+                        .where('categories.id' => category_id).where('status' => :approved)
```

Comment:
This doesn't map correctly to value in database.

WHERE "categories"."id" = $1 AND "resources"."status" = $2        
           [["id", 1], ["status", :approved]]
I, [2017-09-05T02:41:38.886836 #48]  INFO -- : Completed 500 Internal Server Error in 8ms (ActiveRecord: 1.7ms)
F, [2017-09-05T02:41:38.887332 #48] FATAL -- :   
F, [2017-09-05T02:41:38.887367 #48] FATAL -- : ActiveRecord::StatementInvalid (PG::InvalidTextRepresentation: ERROR:  invalid input syntax for integer: "approved"